packages/@okta/i18n/src/properties/login.properties

```diff
@@ -801,6 +801,16 @@ oie.webauthn.error.not.supported = Security key or biometric authenticator is no
 oie.verify.webauth.title = Verify with Security Key or Biometric Authenticator
 oie.verify.webauthn.instructions = You will be prompted to use a security key or biometric verification (Windows Hello, Touch ID, etc.). Follow the instructions to complete verification.
 oie.verify.webauthn.uv.required.instructions = Biometric verification or a PIN is required to sign in with this authenticator.
+oie.verify.webauthn.cant.verify = Can't verify?
+oie.verify.webauthn.cant.verify.biometric.authenticator.title = Are you trying to use a biometric authenticator?
+oie.verify.webauthn.cant.verify.biometric.authenticator.description1 = Biometric authenticators (fingerprint, face recognition, PIN) will only work on the same device on which they were set up.
+oie.verify.webauthn.cant.verify.biometric.authenticator.description2 = If available, set up another security method on the device you used to set up your biometric authenticator.
+oie.verify.webauthn.cant.verify.security.key.title = Are you trying to use a security key?
+oie.verify.webauthn.cant.verify.security.key.description = If you have set up a security key, insert it in a USB port when prompted by the browser and tap on the button or gold disk. Security keys can work on multiple devices.
+oie.verify.webauthn.cant.verify.enrollment.step1 = Open your Okta Dashboard (e.g. yourcompany.okta.com) on the device you used to setup your security key or biometric authenticator
```

Comment:
May be pretty neat to use the actual value for the domain here, since we have it stored in our auth client.